src/components/Details/movie-details.js

```diff
@@ -36,39 +36,61 @@ class Details extends Component {
       <img src={tater} className="tater-logo" alt="logo" />
       </div>
 
+      : !this.state.videos ?
+      <div className='loading'>
+      <h1>LOADING</h1>
+      <img src={tater} className="tater-logo" alt="logo" />
+      </div>
+
       : this.state.movie &&
       <article className='article'>
         <div className='backdrop-container'>
-
-        <img className='backdrop' id={Number(this.state.movie.id)}
-        src={this.state.movie.backdrop_path} alt={`Backdrop of ${this.state.movie.title} movie`}/>
+          <img className='backdrop' id={Number(this.state.movie.id)}
+          src={this.state.movie.backdrop_path} alt={`Backdrop of ${this.state.movie.title} movie`}/>
         </div>
-        <div className='details'>
-          <Link to='/'>
-            <button className='return-btn' >Return to main menu</button>
-          </Link>
-          <h1 className='movie-title'>{this.state.movie.title}</h1>
-          <h3 className='average-rating'>Average Rating: {this.state.movie.average_rating}</h3>
-          <h3 className='release-date'>Release date: {this.state.movie.release_date}</h3>
-
-          <h3>Overview: {this.state.movie.overview}</h3>
-
-          <h3>Genres: {this.state.movie.genres.join(', ')}</h3>
-
-          {this.state.movie.budget > 0 &&
-            <h3>Budget: {this.state.movie.budget}</h3>
-          }
-
-          {this.state.movie.revenue > 0 &&
-          <h3>Revenue: {this.state.movie.revenue}</h3>
-          }
-
-          <h3>Runtime: {this.state.movie.runtime}</h3>
-
-          {this.state.movie.tagline.length > 1 &&
-            <h3>Tagline: {this.state.movie.tagline}</h3>
-          }
+        <div className='details-container'>
+          <div className='details'>
+            <Link to='/'>
+              <button className='return-btn' >Return to main menu</button>
+            </Link>
+            <h1 className='movie-title'>{this.state.movie.title}</h1>
+            <h3><span className="first">Average Rating: </span><span className="second">{(this.state.movie.average_rating).toFixed(2)}</span></h3>
+            <h3><span className="first">Release date: </span><span className="second">{this.state.movie.release_date}</span></h3>
 
+            {this.state.movie.overview.length > 2 &&
```

Comment:
I like how you added conditional rendering to check if the necessary info exists